detox/package.json

```diff
@@ -81,6 +82,10 @@
         "functions": 100,
         "lines": 100
       }
-    }
+    },
```

Comment:
Oops, committed this by mistake, I actually added the the test for them, but forgot to remove the exclusion, will revert it.